actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+module ActionText
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+
+      def install_javascript_dependencies
+        run "rake app:update:bin"
```

Comment:
When this is called running on master, it attempts to overwrite bin/rails and bin/rake with non-spring versions. Why do we need to call this at all? And if we do, how can we make it aware of spring?

Reply:
I think it was added to add `bin/yarn`: https://github.com/rails/rails/pull/35085/commits/028d1611cdbc36b2650f80661cad6c79bb104051

Since Rails 6 already ships with all the necessary bin, I think we can skip it. I tested by creating new application from the master without updating binstub. Works fine. 

Also, on the master, ActionText installer won't work and silently fail because `@rails/actiontext@^6.1.0-alpha` isn't released yet and installer tries to install the latest version.(PR for same: #37824)

Reply:
The addition of this evolved from an issue reported that the install of actiontext would fail silently without yarn installed. In my opinion, I do think it's a good idea to make sure the binstubs exist.

However, I'm not sure why the spring bits are not added. Could it be that `app:update:bin` is not matching what `rails new` generates for binstubs? And if so, is that a conscious decision or can we just update it?

Reply:
@vinistock : I gave it a try by uninstalling yarn from my system on the master branch. 
Running `rails new` **without yarn**  fails with following message.
<img width="1443" alt="yarn-not-installed" src="https://user-images.githubusercontent.com/7736232/81572526-f3b5c000-93c0-11ea-916c-43306da9205b.png">

Running `bundle exec rails action_text:install` **without yarn** also fails. The error is not user friendly but yet it doesn't silently fail.
<img width="1444" alt="fails-without-yarn-on-actiontext-install" src="https://user-images.githubusercontent.com/7736232/81572623-0def9e00-93c1-11ea-80e5-7e6ce2d8348f.png">

I think we should go ahead and remove it and when such case or issue is reported we can rework on it to make it better. Here is the PR to remove it(I am holding that PR since morning, please do close it if not needed): https://github.com/rails/rails/pull/39233

Reply:
I looked into this a bit further. The root of the issue is that we really only need to make sure `bin/yarn` exists, but we're running the `update:bin` generator which tries to override the binstubs generated by spring.

In my opinion, the ideal would be to surface a way of generating the yarn binstub exclusively (e.g.: `app:update:bin:yarn`). That way we wouldn't mess up the binstubs created by spring and yet be able to provide a nice friendly message about yarn not being installed.

That said, I have no objections to removing it until we figure it out appropriately.

Reply:
Let's have something that just ensures the yarn bin is available. Rather than overwriting all the other binstubs.

Reply:
Sounds good. I'll put together a PR and we can iterate on it.